mender/values.yaml

```diff
@@ -10,7 +10,7 @@ global:
   mongodb:
     URL: mongodb://mongodb
   nats:
-    URL: nats://nats:4222
+    URL: nats://mender-nats:4222
```

Comment:
When you set `nats.URL` you're using an external NATS, for e.g. one you deployed manually with a dedicated helm chart. Historically this was named just `nats`, so we kept the same name for backward compatibility: 
https://docs.mender.io/3.2/server-installation/production-installation-with-kubernetes/nats

by updating it, some customer setup could brake. 

If you deploy Mender with the integrated NATS (`nats.enabled: true`) you then have to let `nats.URL: ""`

Reply:
In your own override values files, yes

Reply:
so does the repo not aim to allow a working installation with what's in the repo?
or rather: we provide the included nats chart, but if you use it values or example values file in the repo it won't start - is this intended?

Reply:
Unfortunately yes: historically this helm chart never worked alone, but needed external backing services (NATS, Mongo...). We documented the NATS standalone helm chart setup in the docs (like [this](https://docs.mender.io/3.2/server-installation/production-installation-with-kubernetes/nats)).

When we added the integrated NATS, we could not break the default external NATS and Mongo: customers would have experienced issues with the upgrade.
So the best choice we took was to keep backward compatibility and let the default external NATS.